src/main/resources/application.properties

```diff
@@ -14,6 +15,8 @@ logging.uris.exclude.regex=/health|/info|/ping|/swagger.*|/webjars/.*|/v2/api-do
 azure.application-insights.web.enable-W3C=true
 azure.application-insights.web.enable-W3C-backcompat-mode=true
 
+
+oauth.root.url = http://localhost:9090/auth
```

Comment:
in case I forget to mention it later, whereabouts should have a health indicator for auth too (that calls ping)